ansys/dpf/core/dpf_array.py

```diff
@@ -1,4 +1,5 @@
 """
+
```

Comment:
Again, in my experience this triggers warnings in one of our other tools than the one you are currently looking at.